ostd/src/mm/page_table/mod.rs

```diff
@@ -380,4 +386,43 @@ pub trait PageTableEntryTrait:
     /// The level of the page table the entry resides is given since architectures
     /// like amd64 only uses a huge bit in intermediate levels.
     fn is_last(&self, level: PagingLevel) -> bool;
+
+    /// Converts the PTE into its corresponding `usize` value.
+    fn as_usize(self) -> usize {
+        // SAFETY: `Self` is Copy and has the same memory representation as usize
+        unsafe { transmute_unchecked(self) }
+    }
+
+    /// Converts a usize `pte_raw` into a PTE.
+    ///
+    /// # Safety
+    ///
+    /// The `pte_raw` must represents a valid PTE.
+    unsafe fn from_usize(pte_raw: usize) -> Self {
```

Comment:
If `PageTableEntryTrait` is `Pod`, it means that `PageTableEntryTrait` must be valid for _every_ byte pattern.
 - Then, I don't see why `from_usize` is unsafe.

I believe we should either remove `Pod` or make it safe.